src/authConfig.ts

```diff
@@ -19,24 +19,43 @@ declare global {
     }
 }
 
-const keycloakEnabled = process.env.KEYCLOAK_ENABLED?.toLowerCase() === 'true'.toLowerCase();
+const isEmptyOrWhiteSpace = (input: string | undefined) => {
+    return !input || input.trim() === '';
+}
+
+const keycloakEnabled = (process.env.KEYCLOAK_ENABLED || 'false').toLowerCase() === 'true'.toLowerCase();
+const keycloakLocalUrl = process.env.KEYCLOAK_LOCAL_URL;
+const keycloakIssuer = process.env.KEYCLOAK_ISSUER;
+const serverUrlFromConfig = isEmptyOrWhiteSpace(keycloakLocalUrl) ? keycloakLocalUrl : keycloakIssuer;
```

Comment:
Souldn't the logic here be inverted? Right now, the keycloakLocalUrl is used, only when it's empty or a whitespace